improver_tests/utilities/test_mathematical_operations.py

```diff
@@ -429,7 +429,7 @@ def test_basic(self):
             result.coord("height").points, np.array([10.0, 5.0])
         )
         self.assertArrayAlmostEqual(
-            result.coord("height").bounds, np.array([[10.0, 20.0], [5.0, 10.0]])
+            result.coord("height").bounds, np.array([[20.0, 10.0], [10.0, 5.0]])
```

Comment:
I think I might need some explanation of why the order of the values has changed?